contrib/javaee.wildfly/licenseinfo.xml

```diff
@@ -47,4 +52,26 @@
         <license ref="W3C2" />
         <comment type="CATEGORY_B" />
     </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-app_4_0.dtd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-service_4_0.dtd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss_4_0.dtd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-web_7_3.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-web_4_0.dtd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/datasources_1_4.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-common_6_0.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-as-messaging-deployment_1_0.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/wildfly-messaging-activemq-deployment_1_0.xsd</file>
+        <license ref="LGPL-2.1" />
+        <comment type="CATEGORY_B" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-web_14_1.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-app_8_1.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-common_8_1.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-ejb3-2_1.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-web_14_1.xsd</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
```

Comment:
No they are not donated to ASF but are under ASL 2.0 license they can be incorporated and also because of this license not donation is required right ?

Reply:
I think `ref="Apache-2.0-ASF"` means owned by Apache Software Foundation.

I guess you can use `ref="Apache-2.0"` for stuff under Apache 2.0 owned by somebody else.

Reply:
The "ref" points to the license in the nbbuild/licenses folder. For each license there can the full text be found. In this case "Apache-2.0-ASF" starts with "Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements." That statement would be wrong for foreign code, so the correct variant is "Apache-2.0".

Reply:
ok, fixed